block_designs.tex

```diff
@@ -900,4 +893,136 @@ \section{Steinerovské systémy trojíc, zovšeobecnenia}
 
 \end{theorem_hard}
 
-\TODO konečné jednoduché grupy.
+\section{Symetrické konfigurácie}
+
+\begin{definition}
+	Konfigurácia je množina bodov a priamok, píšeme $(v_\gamma l_\pi)$, kde $v$ je počet bodov, $l$ je počet priamok, $\gamma$ je počet priamok prechádzajúcich každým bodom a $\pi$ je počet bodov na každej priamke.
+\end{definition}
+
+\begin{definition}
+	Konfigurácia je symetrická, ak platí $\gamma = \pi$ (a teda nutne aj $v = l$). Vtedy konfiguráciu píšeme skratene ako $(v_\gamma)$.
+\end{definition}
+
+\begin{definition}
+	Čiastočný Steinerovský systém $S_p(v, k, t)$ je množina $k$-prvkových podmnožín (blokov) $v$-prvkovej množiny bodov taká, že každá $t$-prvková podmonožina bodov sa nachádza v najviac jednom bloku.
+\end{definition}
+
+Konfigurácie $(v_3)$ voláme symetrické $v_3$-konfigurácie.
+
+\begin{theorem}
+	Symetrická $v_3$-konfigurácia je $S_p(v, 3, 2)$.
+\end{theorem}
+
+\begin{proof}
+	Priamo z definície čiastočného Steinerovského systému. Bloky interpretujeme ako priamky. Dva body jednoznačne určujú priamku, preto $t = 2$.
+\end{proof}
+
+\begin{theorem}
```

Comment:
Toto by som nenazýval vetou, keďže ide len o konkrétny prípad. Veta by mala hovoriť o niečom všeobecnom a tiež napovedá, že ide o dôležité tvrdenie. Toto vyzerá skôr ako nejaký konkrétny príklad symetrickej konfigurácie. Dal by som to skôr ako poznámku (remark), lemu alebo hoci aj rovno ako úlohu

Reply:
-> Lemma, ostatné -> Exercise